cmake/onnxruntime_unittests.cmake

```diff
@@ -452,6 +452,14 @@ if(WIN32)
       $<TARGET_FILE_DIR:${test_data_target}>
     )
   endif()
+  if (onnxruntime_USE_OPENVINO)
```

Comment:
remove this for same reason as python.cmake above.
openvino libs should already be in the path so this copy isn't necessary.